yt_dlp/utils.py

```diff
@@ -2340,15 +2340,20 @@ def make_HTTPS_handler(params, **kwargs):
         return YoutubeDLHTTPSHandler(params, context=context, **kwargs)
 
 
-def bug_reports_message():
+def bug_reports_message(before=';'):
     if ytdl_is_updateable():
         update_cmd = 'type  yt-dlp -U  to update'
     else:
-        update_cmd = 'see  https://github.com/yt-dlp/yt-dlp  on how to update'
-    msg = '; please report this issue on https://github.com/yt-dlp/yt-dlp .'
+        update_cmd = 'see <https://github.com/yt-dlp/yt-dlp> on how to update'
```

Comment:
The lack of delimiter is intentional to make it easier for the user to select and copy the URL.

See what happens when you double click on the URL
![image](https://user-images.githubusercontent.com/6623388/115924744-c6f99c00-a49d-11eb-8a64-5514f3b419a1.png)
![image](https://user-images.githubusercontent.com/6623388/115924755-ca8d2300-a49d-11eb-889f-3bda951d1685.png)

Reply:
Works fine for me?

![Double-clicking on a angle-bracket-delimited URL in xfce4-terminal 0.8.10](https://user-images.githubusercontent.com/4331184/115928263-2055d100-a486-11eb-881b-e64ef0e1bcd6.gif)

Reply:
I guess it depends on the console. Many consoles, especially the windows ones don't have special treatment of URLs. Leave space as the delimiter

Reply:
I was tempted to say ‘so much worse for those consoles’, but given how often this tool (or at least the upstream version) ends up being used by non-technical people, I guess we need all the convenience we can manage.